clients/src/main/java/org/apache/kafka/clients/consumer/OffsetResetStrategy.java

```diff
@@ -16,6 +16,13 @@
  */
 package org.apache.kafka.clients.consumer;
 
+import java.util.Locale;
+
 public enum OffsetResetStrategy {
-    LATEST, EARLIEST, NONE
+    LATEST, EARLIEST, NONE;
+
+    @Override
+    public String toString() {
+        return super.toString().toLowerCase(Locale.ROOT);
```

Comment:
I am wondering if this could have an impact when using non-english locale, e.g. there may be places in the code which perform a comparison of the enum with string "latest" but in a different locale this comparison may fail.

Reply:
Quoting the comment above Locale.ROOT
>  This is regarded as the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations.

I think this is better than if we use `toLowerCase(Locale.ENGLISH)`

At the same time, the same method is used for `isolation.level` in `ConsumerConfig`.
```
                                .define(ISOLATION_LEVEL_CONFIG,
                                        Type.STRING,
                                        DEFAULT_ISOLATION_LEVEL,
                                        in(IsolationLevel.READ_COMMITTED.toString().toLowerCase(Locale.ROOT), IsolationLevel.READ_UNCOMMITTED.toString().toLowerCase(Locale.ROOT)),
                                        Importance.MEDIUM,
                                        ISOLATION_LEVEL_DOC)
```